### PR TITLE
Customer Feedback Section modified successfully

### PR DESCRIPTION
**Pull request by tech-dhawal-03**
#11  BUG -  I have successfully worked on my issue and now raising PR...
Features Updated :- 
1. Impressive UI/UX for customer feedback section.
2. Customer Feedback Section name changed to Testimonial.
3. Interactive Carousel with automated sliding.
4. Interactive and colourful user profile card.
5. Responsiveness upto certain extent.

Attaching video also....

https://github.com/user-attachments/assets/dd3c7aa3-d7b3-4904-87a2-eef04a846e19

## Summary by CodeRabbit

- **New Features**
	- Introduced a new customer feedback component, `FeedbackCard`, featuring a carousel to display user reviews.
	- Updated the `ReviewCarousel` component to incorporate the `FeedbackCard` for a simplified testimonial display.

- **Chores**
	- Added new dependencies: `react-slick`, `slick-carousel`, and `daisyui` to enhance carousel functionality and styling.
	- Updated Tailwind CSS configuration to include the DaisyUI plugin for improved component styling.

### CODE REVIEW

**Comment by RamakrushnaBiswal** [comment]
@tech-dhawal-03 resolve the conflicts i will merge

**Comment by tech-dhawal-03** [comment]
I think all conflicts has been resolved...Please let me know if it's creation issue in merging...

**Comment by RamakrushnaBiswal** [comment]
Conflicts are stiil there @tech-dhawal-03

**Comment by RamakrushnaBiswal** [comment]
Make sure it is responsive for all devices

**Comment by tech-dhawal-03** [comment]
Conflicts were already resolved...then to I tried again to do..but I couldn't find any...Please assist me now..I resolved again locally but were not any conflicting files..

**Comment by tech-dhawal-03** [comment]
I don't know why these checks aren't successful. Can you help  me with that ?
Screenshot 2024-10-06 at 11 43 18 AM
These conflicts mentioned in ss, is already resolved.

**Comment by RamakrushnaBiswal** [comment]
Well update your main branch 1st 
Then write code and send PR :)

**Comment by RamakrushnaBiswal** [comment]
@tech-dhawal-03 can you resolve the conflicts please

**Comment by tech-dhawal-03** [comment]
I resolved all conflicts and pushed again after updating into my locals. Please look after it.

**Comment by RamakrushnaBiswal** [comment]
@tech-dhawal-03 stll some conflicts

File: src/Components/Cards/FeedbackCard.jsx [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+        rating: 4,
+    },
+    {
+        name: "Ethan White",
+        review:
+            "The vibe is amazing, and the staff is super friendly. Highly recommend!",
+        img: "https://randomuser.me/api/portraits/men/33.jpg",
+        rating: 5,
+    },
+];
```
[the remark this answers is not in the document]
Comment (tech-dhawal-03) [reply src/Components/Cards/FeedbackCard.jsx:80]: But this is not required now. The website I got to work on have already comments defined in array. So, I don't think so dynamic data fetching is required very now.

**Pull request closed without merging**